e2e/test/helpers/templates/FaultInjection.cs

```diff
@@ -212,7 +212,7 @@ public static bool FaultShouldDisconnect(string faultType)
                     {
                         VerboseTestLogger.WriteLine($"{nameof(FaultInjection)}: Performing test operation for device - Run {counter++}.");
                         await testOperation(deviceClient, testDevice).ConfigureAwait(false);
-                        await Task.Delay(TimeSpan.FromSeconds(1)).ConfigureAwait(false);
+                        await Task.Delay(TimeSpan.FromSeconds(2)).ConfigureAwait(false);
```

Comment:
Would it be worth having some kind of constant for this instead of manually changing it everywhere?